### DISCUSSION TITLE
external storage mounting to wrong partition

### DISCUSSION THREAD

**Discussion by 69fordf100** [outdoorbits/little-backup-box#161] [Q&A]
I'm having an issue where my external drive is mounting to the small partition(sdb2) and I cannot figure out how to mount to the larger partition(sdb1)
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sdb           8:16   0 232.9G  0 disk 
|-sdb1        8:17   0 232.4G  0 part 
`-sdb2        8:18   0   519M  0 part /media/storage
sdc           8:32   1 119.1G  0 disk 
`-sdc1        8:33   1 119.1G  0 part /media/source
mmcblk0     179:0    0   7.4G  0 disk 
|-mmcblk0p1 179:1    0   256M  0 part /boot
`-mmcblk0p2 179:2    0   7.1G  0 part /

**Comment by 69fordf100** [comment]
Ive figured out that was a windows recovery partition left over from an old pc and have since removed it. i would still like to know in an event such as this though, what could be done to change to the other partition?

**Comment by outdoorbits** [comment]
LBB always mounts the first suitable (ext or fat) partition of a device. If you really need multi partition devices on traveling, just prepare them this way.
This automatism is done to enable LBB to work as robust and predictable as possible, even in headless situations. Configuring other partitions would cost flexibility in exchanging the target device.
